plugins/modules/helm.py

```diff
@@ -735,6 +735,8 @@ def main():
                     chart_version,
                     replace,
                 )
+                if would_change:
```

Comment:
```suggestion
                if would_change and module._diff:
```

Reply:
Ansible takes care of displaying the diff or not, so the fact to pass the diff or not here is not changing the output.
So in order to keep it simple, I didn't add this condition. Is that ok?

Reply:
Unfortunately, without this the diff will still display if the task has `diff: no` and `--diff` was passed on the command line.